hbase-server/src/test/java/org/apache/hadoop/hbase/client/TestFromClientSide.java

```diff
@@ -147,6 +148,47 @@ public void testDuplicateAppend() throws Exception {
     }
   }
 
+  /**
+   * Test batch append result when there are duplicate rpc request.
+   */
+  @Test
+  public void testDuplicateBatchAppend() throws Exception {
+    HTableDescriptor hdt = TEST_UTIL
+      .createTableDescriptor(name.getTableName(), HColumnDescriptor.DEFAULT_MIN_VERSIONS, 3,
```

Comment:
Use relevant method for createModifyableTableDescriptor? Given that HTableDescriptor is deprecated.